pkg/codec/encodings/type_codec.go

```diff
@@ -138,6 +149,29 @@ func decode(c map[string]TypeCodec, raw []byte, into any, itemType string, exact
 	return codec.Convert(reflect.ValueOf(val), reflect.ValueOf(into), nil)
 }
 
-func (c CodecFromTypeCodec) GetMaxDecodingSize(ctx context.Context, n int, itemType string) (int, error) {
-	return c.GetMaxEncodingSize(ctx, n, itemType)
+func getCodec(c map[string]TypeCodec, itemType string) (TypeCodec, error) {
+	// itemType could recurse into nested structs
+	path := extendedItemType(itemType)
+
+	// itemType could recurse into nested structs
```

Comment:
Did you mean to write the same comment twice here?

Reply:
Ooops I didn't mean to do that.